swagger/prime.yaml

```
@@ -347,6 +347,60 @@ paths:
       description: Gets entitlements
       operationId: getPrimeEntitlements
       summary: Gets entitlements for a move order by ID
+  '/move_task_orders/{moveTaskOrderID}/mto_shipments/{mtoShipmentID}':
```

Can we change the name of the endpoint to be consistent to the other endpoints naming scheme
```suggestion
  '/move-task-orders/{moveTaskOrderID}/mto-shipments/{mtoShipmentID}':
```